src/pages/user/list.jsx

```diff
@@ -24,23 +40,56 @@ export const UserList= () => {
     }, []);
 
     if(loading) {
-        return (<PageLoading />)
+    return (<PageLoading />)
     }
 
+    let processedUserList = users
+        .filter((user) => 
+            user.uuid.toLowerCase().includes(search) ||
+            user.firstname.toLowerCase().includes(search) || 
+            user.lastname.toLowerCase().includes(search) ||
+            user.username.toLowerCase().includes(search) ||
+            user.email.toLowerCase().includes(search) ||
+            user.phone.includes(search)
+        )
+        .sort(SORTING_TYPES[activeSortingMethod])
+
     return (
         <>
             <DashboardHeader border>
                 <DashboardTitle>
                     Brukeradministrasjon
                 </DashboardTitle>
                 <DashboardSubtitle>
-                    {users.length} brukere registrert
+                    {users.length} brukere registrert{search ? <> – Viser for øyeblikket {processedUserList.length} {processedUserList.length === 1 ? "bruker" : "brukere"}</> : null}
```

Comment:
Isn't "Viser XX av XX registrerte brukere" more natural?

Reply:
I agree, I changed the message!